tumuli/urls.py

```diff
@@ -16,14 +16,28 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.conf import settings
+from django.conf.urls import url, include
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+from biography import views
+
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/users/', views.UserList.as_view()), # get a list off all users
```

Comment:
This is a performance & security risk. Let's remove this and assume the the client knows the username.